cpp/DataStorm/stock/CMakeLists.txt

```diff
@@ -4,6 +4,10 @@ project(datastorm_stock CXX)
 
 include(../../cmake/common.cmake)
 
+# This demo requires C++20
+set(CMAKE_CXX_STANDARD 20)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
```

Comment:
I would keep it C++17 compatible, using std::ranges is not that important for the demo.

Reply:
We need c++20 for other parts of this demo, namely it was already using designated initializer syntax. I think its fine for a few demos to use c++20

Reply:
I guess is fine if it can be build in all supported platforms.